src/content/platform-integration/ios/setup-ios.md

```diff
@@ -0,0 +1,272 @@
+If you don't have one, create one.
+You must also enroll in the [Apple Developer][] program
+and create an [Apple Developer account][]. If you
+are just _testing_ your app on an iOS device,
+a personal Apple Developer account is free and works.
+
+:::note Apple Developer program
+When you want to _deploy_ your app to the
+Play store, you will need to upgrade your
+personal Apple Developer account to
+a professional account.
+:::
+
+### Prepare the device
+
+Find your **Profiles & Device Management** menu under
+**Settings**. (The exact location can vary by device.)
+Toggle your certificate to **Enable**.
+
+:::note
+If you can't find **VPN & Device Management** menu,
+run your app on your iOS device once, then try again.
+:::
+
+1. Under the **Developer App** heading,
+   you should find your certificate.
+
+1. Tap the certificate.
+
+1. Tap **Trust "&lt;certificate&gt;"**.
+
+1. When the dialog displays, tap **Trust**.
+
+If the **codesign wants to access key...** dialog appears:
+
+1. Enter your macOS password.
+
+1. Tap **Always Allow**.
+
+[Trust Mac]: /assets/images/docs/setup/trust-computer.png
+[Developer Mode]: {{site.apple-dev}}/documentation/xcode/enabling-developer-mode-on-a-device
+[Apple Developer]: {{site.apple-dev}}/programs/
+[Apple Developer account]: {{site.apple-dev}}/account
+
+{% endtab %}
+{% endtabs %}
+
+---
+
+## Install CocoaPods
+
+If your app depends on any [Flutter plugin][]
+that uses native iOS code,
+install [CocoaPods][cocoapods].
+CocoaPods bundles various dependencies that
+support connecting Flutter to native iOS code.
+
+1. Install `cocoapods` following the
+   [CocoaPods install guide][cocoapods].
+
+   ```console
+   $ sudo gem install cocoapods
+   ```
+
+1. Launch your preferred text editor to update its path variable.
+
+1. For example, if you use Zsh, open `~/.zshenv` in your text editor.
+
+1. Copy the following line and paste it at the end of your `~/.zshenv` file.
+
+   ```bash
+   export PATH=$HOME/.gem/bin:$PATH
+   ```
+
+1. Save your `~/.zshenv` file.
+
+1. To apply this change, restart all open terminal sessions.
+
+[cocoapods]: https://guides.cocoapods.org/using/getting-started.html#installation
+[Flutter plugin]: /packages-and-plugins/developing-packages#types
+
+## Check your development setup with Flutter doctor
+
+The `flutter doctor` command validates all required components of a
+complete Flutter development environment.
+
+From the command-line, run the following command:
+
+   ```console
+   $ flutter doctor
+   ```
+
+Since these instructions did not install for Android
+or desktop development, you can ignore Flutter doctor's
+error reports for those components for now.
+
+If the `flutter doctor` command returns other errors,
+run it again with the verbose flag.
+
+```console
+$ flutter doctor -v
+```
+
+Follow the instructions in the output for software
+you might need to install or further tasks to perform.
+
+**Congratulations.**
+You can now start developing Flutter apps for iOS!
```

Comment:
I have some infrastructure work planned to improve this next step section for the Android version of this page. Once we land that, we can update this section to use it as well.